components/Utils/src/formatDate.tsx

```diff
@@ -22,8 +32,8 @@ export const formatDate = ({
   dateTime,
   now = new Date().toISOString(),
   relative,
-  locale = nl,
-  format = relative ? 'd MMMM yyyy' : 'd-M-yyyy',
+  locale = 'nl-NL',
```

Comment:
Kunnen we hier niet gewoon de local uit de context halen? 

`const { currentLocale } = useContext(LocaleContext);`

Reply:
Je wilt niet de locale hier ophalen uit de context, omdat je niet zeker weet in welke applicatie dit gebruikt wordt. Dit zou ook buiten de NL portal gebruikt moeten kunnen worden, dus vandaar dat ik het via de prop doe.

Reply:
Na overleg heb ik de default locale gewijzigd naar de browser locale, en gaan we in de toekomst mogelijk een keer kijken of we een generiekere oplossing voor dates/locales kunnen maken.